app/javascript/controllers/dropzone_controller.js

```diff
@@ -5,24 +5,25 @@ export default class extends Controller {
     static targets = ["progress"]
 
     connect() {
-        console.log("Dropzone connected", this.element.id);
         this.dropzone = new Dropzone(this.element, {
             paramName: "content[files]",
             uploadMultiple: true,
             createImageThumbnails: false,
             addRemoveLinks: false,
             disablePreviews: true,
+            maxFilesize: 10000, // 10GB
         });
+        this.progress = 0;
         this.dropzone.on("addedfiles", () => {
-            console.log('addedfiles');
             this.progressTarget.classList.remove("d-none");
+            // This shows the user that something is going on since there may be a paused before first progress.
+            this.updateProgress(2);
```

Comment:
Why `2`?

Reply:
Arbitrary. Large enough to be noticed, but small enough not to be deceptive.